sklearn/ensemble/_base.py

```diff
@@ -71,7 +70,7 @@ def _set_random_states(estimator, random_state=None):
     to_set = {}
     for key in sorted(estimator.get_params(deep=True)):
         if key == 'random_state' or key.endswith('__random_state'):
-            to_set[key] = random_state.randint(MAX_RAND_SEED)
+            to_set[key] = random_state.randint(MAX_INT_32)
```

Comment:
@ogrisel Here we have the same issue that `MAX_INT32` is not included. However, if we make a change, we will have a behaviour change. Shall we let it how it is?

Reply:
+1 for no behavior change.